ces-poc-demo/WeatherForecast.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace ces_poc_demo
+{
+    public class WeatherForecast
+    {
+        [Key]
+        public int ID { get; set; }
+
+        [Required]
+        [Range(-270, 270)]
```

Comment:
Please add tests for all the validations as well

Reply:
added tests for the validations